api/graphql/query/User/availability.js

```diff
@@ -1,8 +1,11 @@
 const availability = async (parent, { dateTimeRange }, { loaders }) =>
   loaders.userAvailability.load({
     user_id: parent.id,
-    dateTimeRange,
-    exculdeEvents: [],
+    dateTimeRange: {
+      start_at: new Date(dateTimeRange.start_at),
+      end_at: new Date(dateTimeRange.end_at),
+    },
```

Comment:
Idk why but `dateTimeRange.start_at` is string in this parse. Had to parse it to `dateObject` for proper comparing.

Reply:
Because that is how the custom scalar graphql type for DateTime serializes and deserializes it as.